src/hypertable.c

```diff
@@ -247,7 +247,7 @@ ts_hypertable_from_tupleinfo(const TupleInfo *ti)
 
 	ts_hypertable_formdata_fill(&h->fd, ti);
 	h->main_table_relid =
-		ts_get_relation_relid(NameStr(h->fd.schema_name), NameStr(h->fd.table_name), false);
+		ts_get_relation_relid(NameStr(h->fd.schema_name), NameStr(h->fd.table_name), true /*FIXME*/);
```

Comment:
actually... it seems like this can be an `InvalidOid` 

as drop is handling it:

https://github.com/timescale/timescaledb/blob/4dce87a1c4a53d080676bfff042e24aa5b822cec/src/hypertable.c#L705